src/Files.App/Extensions/KeyboardAcceleratorExtensions.cs

```diff
@@ -8,10 +8,15 @@ public static class KeyboardAcceleratorExtensions
 	{
 		public static bool CheckIsPressed(this KeyboardAccelerator keyboardAccelerator)
 		{
-			return Microsoft.UI.Input.InputKeyboardSource.GetKeyStateForCurrentThread(keyboardAccelerator.Key).HasFlag(CoreVirtualKeyStates.Down) && // check if the main key is pressed
-				(!keyboardAccelerator.Modifiers.HasFlag(VirtualKeyModifiers.Menu) || Microsoft.UI.Input.InputKeyboardSource.GetKeyStateForCurrentThread(VirtualKey.Menu).HasFlag(CoreVirtualKeyStates.Down)) && // check if menu (alt) key is a modifier, and if so check if it's pressed
-				(!keyboardAccelerator.Modifiers.HasFlag(VirtualKeyModifiers.Shift) || Microsoft.UI.Input.InputKeyboardSource.GetKeyStateForCurrentThread(VirtualKey.Shift).HasFlag(CoreVirtualKeyStates.Down)) && // check if shift key is a modifier, and if so check if it's pressed
-				(!keyboardAccelerator.Modifiers.HasFlag(VirtualKeyModifiers.Control) || Microsoft.UI.Input.InputKeyboardSource.GetKeyStateForCurrentThread(VirtualKey.Control).HasFlag(CoreVirtualKeyStates.Down)); // check if ctrl key is a modifier, and if so check if it's pressed
+			return
+				// Check if the main key is pressed
```

Comment:
Should the comments be moved to the top of the method? It looks a little messy right now.

Reply:
@d2dyno1 What do you think? 

If one line is long, the comment will not be visible fully, so I think I should add a line for the comment.

Reply:
The comment comments can be split onto multiple lines, we should just move it higher up.